client/packages/invoices/src/OutboundShipment/api/operations.generated.ts

```diff
@@ -345,6 +353,24 @@ export const InvoiceCountsDocument = gql`
   }
 }
     `;
+export const BarcodesDocument = gql`
+    query barcodes($storeId: String!, $filter: BarcodeFilterInput) {
+  barcodes(storeId: $storeId, filter: $filter) {
+    ... on BarcodeConnector {
+      __typename
+      nodes {
+        id
+        itemId
+        manufacturerId
+        packSize
+        parentId
```

Comment:
i like the idea of just querying for what's needed/used (one advantage of graphql)

Reply:
oops this is meant to be for client/packages/invoices/src/OutboundShipment/api/operations.graphql